README.md

```diff
@@ -59,6 +59,25 @@ A simple helper to perform async application initialization for the generic host
     }
     ```
 
+    You can also pass a `CancellationToken` in order to propagate notifications to cancel the initialization if needed.
+
+    In the following example, the initialization will be cancelled:
```

Comment:
This sentence looks like it's missing something. You should probably mention what would cause it to be cancelled.
(sorry I didn't catch it in my first review...)